.env.example

```diff
@@ -0,0 +1,4 @@
+# create your own .env file in the root of the projekt
+
+# this variable has to be set when you are developing in a docker container 
+# POSTGRESQL_SERVICE_HOST=hibernate
```

Comment:
Setting this env is okay, but you have to mention the networking part as well.

If you just set this env, an quarkus dev container won't find the db with the hostname "hibernate".

Docker container are not connected by default. There are two solutions available

1. repalce "hibernate" with the host IP
2. add networking commands to the deployment of the postgres DB container

Additionally, move your comment "how to run postgres DB" into the readme or the .env.example to make this information general available

Reply:
> The quarkus dev container and the database have to be in the same network to resolve hostnames, therefore do the following steps:
> 
> * `docker network create --driver bridge quarkus-net`
> * attach already created container to the new network: `docker network connect quarkus-net <container-name>`
>   **or** create new container in the network: `docker build --build-arg GH_TOKEN=<GH-TOKEN> --build-arg username=<git-username>--build-arg email=<git-email> -t quarkus_image .` (unchanged) + `docker run --network quarkus-net --name quarkus -it quarkus_image`
> * attach database container to the network: `docker run --rm=true --name hibernate --network quarkus-net -e POSTGRES_USER=hibernate -e POSTGRES_PASSWORD=hibernate -e POSTGRES_DB=hibernate -p 5432:5432 postgres:14.1`

I have already added the networking commands. Therefore I think that the containers are connected via the quarkus-net (in my dev container it worked anyway.)

Reply:
Yes, you are right. The containers are connected and your set up is working.

But, where is the `docker commands` documentation ?

The comments of this issue won't be accessible for future users of your respository. They need a section in your LaTex documentation or a section in a readme.md file or somethin else to get the necessary information to get the project running.

Reply:
Okay, now I understand.

Reply:
I will open a new PR for the corresponding adjustments in the latex docs.